apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -0,0 +1,145 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import color from '@cdo/apps/util/color';
+import javalabMsg from '@cdo/javalab/locale';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
+import {ViewType} from '@cdo/apps/code-studio/viewAsRedux';
+import CodeReviewDataApiV2 from './codeReview/CodeReviewDataApiV2';
+import ReviewNavigator from './codeReview/ReviewNavigator';
+import Button from '@cdo/apps/templates/Button';
+
+export const VIEWING_CODE_REVIEW_URL_PARAM = 'viewingCodeReview';
+
+const CommitsAndReviewTab = ({
+  onLoadComplete,
+  channelId,
+  serverLevelId,
+  serverScriptId,
+  viewAsCodeReviewer,
+  viewAsTeacher,
```

Comment:
I'm not sure what the best way to tackle this is but it seems like this component can be viewed in three distinct modes: author, teacher, and code reviewer.  It would be nice for this to be a bit more explicit and easier to understand what the differences are between the three modes (even if it's just adding some comments).

Reply:
I'm going to try to make the cases clear in the code and tests, but will add comments if necessary!